### ISSUE TITLE
Error when escaping single quotes in comments

### ISSUE THREAD

**Issue by doflo-dfa** [notiz-dev/prisma-dbml-generator#26]
I am using multiple generators (https://github.com/unlight/prisma-nestjs-graphql) against my prisma file and it uses comments to map scalers (and other concepts) unfortunately it looks like this is causing some problems. It looks like only the first instance of special characters in comments are being replaced  for instance the following line in my prisma file

```
/// @FieldType({ name: 'Scalars.GraphQLBigInt', from: 'graphql-scalars', input: true, output: true })
serial BigInt
```
will be transformed into this in the dbml output

```
serial BigInt [not null, note: '@FieldType({ name: \'Scalars.GraphQLBigInt', from: 'graphql-scalars', input: true, output: true })']
```

**Comment by doflo-dfa** [comment]
https://github.com/notiz-dev/prisma-dbml-generator/pull/27

**Comment by marcjulian** [comment]
@doflo-dfa thanks for the issue and the pr! Do you mind adding a test as well?

**Comment by marcjulian** [comment]
It is fixed in [v0.8.3](https://github.com/notiz-dev/prisma-dbml-generator/releases/tag/v0.8.3)!
